INSTALL.md

```diff
@@ -1,5 +1,4 @@
 Building Bitcoin
 ================
 
-See doc/build-*.md for instructions on building the various
-elements of the Bitcoin Core reference implementation of Bitcoin.
+See <a href="https://github.com/bitcoin/bitcoin/tree/master/doc">doc/build-</a>*.md for instructions on building the various elements of the Bitcoin Core reference implementation of Bitcoin.
```

Comment:
- Please use a repo-relative link instead
- Please use markdown link syntax
e.g. something like this should work, i think
```[doc/build-\*.md](/doc)```